### PR TITLE
Added HW 4.1 Solution

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 1]
```diff
@@ -8,6 +8,14 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException();
+        }
+        try {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/SqrtOperation.java]: do we need this try-catch block?

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 2]
```diff
@@ -8,6 +8,14 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/SqrtOperation.java]: pass message here

File: src/main/java/core/basesyntax/SqrtOperation.java (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/SqrtOperation.java]: Got it. Fixed.

File: src/main/java/core/basesyntax/SqrtOperation.java (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/SqrtOperation.java]: Fixed.

**Review by Sofasmile — APPROVED** [review]
Good)

**Pull request closed without merging**